usr/share/rear/rescue/GNU/Linux/400_use_serial_console.sh

```diff
@@ -1,4 +1,7 @@
 
+DebugPrint "skipping usr/share/rear/rescue/GNU/Linux/400_use_serial_console.sh"
```

Comment:
Better remove this file if no longer needed, IMHO

Reply:
See my commit message in
https://github.com/rear/rear/pull/2961/commits/2c5875d9bb32fa0b3c99e590f83007b326d10743
(excerpt)
```
... later this script should be
completely removed provided things work
```